sonar-java-plugin/pom.xml

```diff
@@ -49,22 +49,11 @@
     </dependency>
 
     <dependency>
-      <groupId>org.codehaus.sonar</groupId>
-      <artifactId>sonar-java-api</artifactId>
-      <scope>provided</scope>
-    </dependency>
-    <dependency>
-      <groupId>org.codehaus.sonar</groupId>
+      <groupId>org.sonarsource.sonarqube</groupId>
       <artifactId>sonar-plugin-api</artifactId>
       <scope>provided</scope>
     </dependency>
```

Comment:
@benzonico but ultimately usage of `slf4j-api` should be removed - see https://jira.sonarsource.com/browse/SONAR-5700